### DISCUSSION TITLE
Force '/index.html' in internal links

### DISCUSSION THREAD

**Discussion by stef41** [hbstack/.github#57] [Q&A]
How to systematically append '/index.html' for each link generated internally ? It would be super useful for static hosting like in aws s3

**Comment by razonyang** [comment]
See https://discourse.gohugo.io/t/how-to-force-index-html-in-all-a/5059 and https://gohugo.io/getting-started/configuration/#uglyurls.

**Comment by stef41** [comment]
Thanks I added `uglyURLs: true` in hugo.yaml and it worked well for almost all links.
It just doesn't work with the language selector, for example it I click on 'deutschland' as specified in my languages.yaml it still redirects to '/deutschland/' and not '/deutschland/index.html'
![Screenshot from 2024-05-19 18-04-48](https://github.com/hbstack/.github/assets/87280280/b8ac3687-c626-42b7-8156-5822d5fd7093)

**Reply by razonyang** [reply]
Please track this on https://github.com/gohugoio/hugo/issues/7497.
